base/applications/notepad/settings.c

```diff
@@ -228,8 +236,18 @@ void NOTEPAD_SaveSettingsToRegistry(void)
 {
     HKEY hKey;
     DWORD dwDisposition;
+    WINDOWPLACEMENT wp;
+    UINT x, y, cx, cy;
+
+    wp.length = sizeof(wp);
+    GetWindowPlacement(Globals.hMainWnd, &wp);
+    x = wp.rcNormalPosition.left;
+    y = wp.rcNormalPosition.top;
+    cx = wp.rcNormalPosition.right - x;
+    cy = wp.rcNormalPosition.bottom - y;
+    if (wp.flags & WPF_RESTORETOMAXIMIZED)
+        cy |= 0x80000000;
```

Comment:
I think you may have indirectly answered the question in your previous comment https://github.com/reactos/reactos/pull/5806/files#r1365949076 , but:
is this 0x80000000 flag the trick to save in the registry the "maximized" (versus "normal") window state?

Does this mean that before your PR, our notepad didn't save that maximized state?

Reply:
Yes. Try the `start /wait ...` command I posted above.

Reply:
Thanks, this is indeed what I suspected.